### PR TITLE
chore(starter): upgrade to Angular 8

### PR DESCRIPTION
**Pull request by nicanac**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/NationalBankBelgium/stark/blob/master/CONTRIBUTING.md#-commit-message-guidelines
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Currently using ng7, and must be aligned with the version 8

Issue Number: #2565 

## What is the new behavior?
Implementing ng 8 on the starter

## Does this PR introduce a breaking change?
```
[X] Yes
[ ] No
```

## Other information

### CODE REVIEW

File: starter/package.json [hunk 1]
```diff
@@ -112,51 +112,52 @@
...
-    "@types/node": "~10.17.13",
-    "@types/webpack-env": "~1.15.0",
-    "cross-env": "~7.0.0",
+    "@nationalbankbelgium/stark-build": "file:../dist/packages-dist/stark-build/nationalbankbelgium-stark-build-10.2.0-a92e5a34.tgz",
+    "@nationalbankbelgium/stark-testing": "file:../dist/packages-dist/stark-testing/nationalbankbelgium-stark-testing-10.2.0-a92e5a34.tgz",
+    "@types/core-js": "~2.5.4",
+    "@types/hammerjs": "~2.0.39",
+    "@types/node": "~10.17.54",
+    "@types/webpack-env": "~1.16.0",
+    "browserstack-local": "~1.4.8",
```
Comment (SuperITMan) [inline starter/package.json:152]: Is it required to add "browserstack-local" ?
As we don't have any e2e tests for the starter, could you maybe remove it ? 😊

File: starter/src/index.html [hunk 2]
```diff
@@ -4,30 +4,18 @@
...
 		<!-- necessary for HTML5 History API which adds support for pretty URLs (see app.ts) -->
 		<meta charset="utf-8" />
-		<title><%= htmlWebpackPlugin.options.starkAppMetadata.name %></title>
 
-		<meta name="description" content="<%= htmlWebpackPlugin.options.starkAppMetadata.description %>" />
-
-		<% if (webpackConfig.htmlElements.headTags) { %>
-		<!--Configured Head Tags  -->
-		<%= webpackConfig.htmlElements.headTags %> <% } %>
+		<title><%= .starkAppMetadata.name %></title>
```
Comment (SuperITMan) [inline starter/src/index.html:8]: There is a missing word here, it should be: 

```html
<%= starkOptions.starkAppMetadata.name %>
```

**Review by SuperITMan — CHANGES_REQUESTED** [review]
2 small changes to apply then we can merge 😊 

Good job 👍

File: starter/src/index.html (hunk 2 above)
Comment (nicanac) [reply starter/src/index.html]: Opps solved :-)

File: starter/package.json (hunk 1 above)
Comment (nicanac) [reply starter/package.json]: Yeah true, but I thought, it would be (in the future) a push to make the projects use it. At least a bit :-).
It's removed.

**Review by SuperITMan — APPROVED** [review]
LGTM 👍

**Pull request merged**